edfi-repo-auditor/README.md

```diff
@@ -0,0 +1,24 @@
+# Ed-Fi Repo Auditor
+
+Python script to get a report of the compliance of the repositories to the checklist provided in [Repository Scoring](https://techdocs.ed-fi.org/pages/viewpage.action?spaceKey=EDFIODS&title=Repository+Scoring).
+
+The file [scoring.json](./scoring.json) provides the guidelines to be followed. The property names are in the same format that the code will follow to assign a value. This file provides a value for each property, this file can be modified accordingly. Additionally, the `threshold` property specifies the limit that will be tolerated to indicate if the repository needs to make adjustments according to the scoring.
+
```

Comment:
Please rewrap the lines above so that they're easier to read. I like to use the VS Code `rewrap` extension, which will wrap these at 80 characters.